deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -614,6 +616,8 @@ def insert_data_redhat_feed(data, field_name, field_value):
         data['replace_me'] = field_value
         string_data = json.dumps(data, indent=4, ensure_ascii=False).replace('"replace_me"', f"{field_name}")
 
+    string_data += ',\n' + json.dumps(append_data, indent=4, ensure_ascii=False)
+
```

Comment:
It is better to use fstrings rather than `+` operator.

Reply:
Done in f3a1c5d2899020f0620c96c8a9cf2d596245f804